internal/app/server/server.go

```diff
@@ -164,7 +164,7 @@ func RunWithContext(ctx context.Context, cancel context.CancelFunc, bootstrapCon
 				stopMutex.Unlock()
 			}
 		case sig := <-sigs:
-			logger.Info(ctx, "received interrupt signal:", sig.String())
+			logger.Info(ctx, "received interrupt signal: %s", sig.String())
```

Comment:
Thanks for the fix. Could you use `logger.With("signal", sig.String()).Info(ctx, "received interrupt signal")`?

It allows logging engines (ex: Kibana) to filter on the exact message and context cues.

Reply:
Sure, updated. Thank you!